src/main/java/MorseCodeConverter.java

```diff
@@ -0,0 +1,121 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class MorseCodeConverter {
+    Map<String, String> mapToMorse = new HashMap<>();
+    Map<String, String> mapToEnglish = new HashMap<>();
+
+    public MorseCodeConverter(){
+        fillMapToMorse(mapToMorse);
+        fillMapToEnglish(mapToEnglish);
```

Comment:
Because these are now member variables, they don't need to be passed into the method.